src/interfaces/lightspeed.ts

```diff
@@ -67,11 +71,17 @@ export interface IssueFeedbackEvent {
   description: string;
 }
 
+export interface PlaybookOutlineEvent {
+  action?: ThumbsUpDownAction;
```

Comment:
I cannot find a case where we would want these parameters to be optional. We can just flag them both as mandatory.

Reply:
Changed these two required parameters in the latest commit